App.js

```diff
@@ -19,72 +19,52 @@ import {
     AppLogo,
     AppDesc,
     StyledHomeButtonView,
-    StyledHomeText
+    StyledHomeText,
```

Comment:
should remove this since u removed the component

Reply:
Good catch, will do :)

Reply:
Will I have to make an entire new PR for this change? I just pushed the changes I made where I deleted the component but I don't see it reflected here

Reply:
nope. can just make it here since u remove it in this pr